linkerd2-proxy/src/main.rs

```diff
@@ -111,12 +113,15 @@ fn main() {
         let drain = app.spawn();
         tokio::select! {
             _ = signal::shutdown() => {
-                info!("Received shutdown signal");
+                info!(grace_period = ?shutdown_grace_period, "Received shutdown signal");
             }
             _ = shutdown_rx.recv() => {
-                info!("Received shutdown via admin interface");
+                info!(grace_period = ?shutdown_grace_period, "Received shutdown via admin interface");
```

Comment:
I'd prefer that we either log the grace period here or below in the timeout case, but it seems a bit overzealous to log it in both places. I'd suggest omitting it here since it won't be encountered the vast majority of the time.